packages/node-sdk/src/client.ts

```diff
@@ -539,13 +536,9 @@ export class BucketClient {
    **/
   public async updateCompany(
     companyId: string | number,
-    opts?: TrackOptions & { userId?: string },
+    opts?: TrackOptions & { userId?: string | number },
```

Comment:
Still think we can make a type for these instead of repeating it.

Reply:
done 👍